lib/implore.es6.js

```diff
@@ -121,7 +121,16 @@ export default function implore(request, callback) {
 			response.body = xhr.responseText;
 			response.status = xhr.status;
 			response.type = xhr.getResponseHeader('Content-Type');
-			response.headers = xhr.getAllResponseHeaders();
+			const rawHeaders = xhr.getAllResponseHeaders();
+			// Just in case some headers don't get processed properly
+			response.rawHeaders = rawHeaders;
+			// Processed headers as a hash for easier use
+			const responseHeaders = rawHeaders.split("\n");
+			response.headers = {};
+			for (let header of responseHeaders) {
+				const parts = header.split(': ');
```

Comment:
Just as a note to myself and others, it looks like the space isn't optional when it comes to the string that is actually returned by `getAllResponseHeaders`; see 3-1 in section 4.6.5 here: https://xhr.spec.whatwg.org/#the-getallresponseheaders()-method